multimedia/templates/multimedia/book_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "multimedia/book_base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+{{ block.super }}
```

Comment:
So is it the common practice then? If so, then I will add it to all subsequent blocks